src/cascadia/WindowsTerminal/icon.cpp

```diff
@@ -27,22 +27,27 @@ static int _GetActiveAppIconResource()
     return iconResource;
 }
 
-void UpdateWindowIconForActiveMetrics(HWND window)
+HANDLE GetActiveAppIconHandle(int size)

```

Comment:
I believe there's only an `ICON_SMALL` and an `ICON_BIG` (https://docs.microsoft.com/en-us/windows/win32/winmsg/wm-seticon). 

Unsure which one to fall back to though, as long as there is a version that it does fallback to maybe it's not too big of a deal? Perhaps it would be better to make it a `bool small` and `true=ICON_SMALL` and `false=ICON_BIG`. Then there's no room for error?